src/cdk/listbox/listbox.ts

```diff
@@ -418,6 +418,19 @@ export class CdkListbox<T = unknown> implements AfterContentInit, OnDestroy, Con
   /** Whether the listbox currently has focus. */
   private _hasFocus = false;
 
+  /** A reference to the option that was active before the listbox lost focus. */
+  private _previousActiveItem: CdkOption<T> | null = null;
+
+  constructor() {
+    fromEvent(window, 'blur')
+      .pipe(takeUntil(this.destroyed))
+      .subscribe(() => {
```

Comment:
Can we move this into a separate class member so we don't have so much logic in the constructor?

Reply:
Done